lang/funcs/collection.go

```diff
@@ -257,10 +257,8 @@ var CompactFunc = function.New(&function.Spec{
 	},
 })
 
-// ContainsFunc contructs a function that determines whether a given list
-// contains a given single value as one of its elements.
-//
-// ContainsFunc also works on sets, as HCL automatically converts sets to lists.
+// ContainsFunc contructs a function that determines whether a given list or
```

Comment:
If we're updating the docs here anyhow, contructs => constructs?

Reply:
nice catch!

Reply:
there were 18 of those 🙃 thanks copy pasta!